src/WireFormat.h

```diff
@@ -148,6 +154,17 @@ enum ControlOp {
 };
 
 /**
+ * Used in linearizable RPCs to check whether or not the RPC can be processed.
+ */
+struct ClientLease {
+    uint64_t leaseId;           /// A cluster unique id for a specific lease.
+    uint64_t leaseTerm;         /// Cluster time after which the lease may have
+                                /// become invalid.
+    uint64_t timestamp;         /// Cluster time when this lease information was
```

Comment:
The name "timestamp" conveys almost no useful information. How about "leaseStartTime"?